00-RELEASENOTES

```diff
@@ -1,16 +1,165 @@
-Hello! This file is just a placeholder, since this is the "unstable" branch
-of Valkey, the place where all the development happens.
+Valkey 8.1 release notes
+========================
+--------------------------------------------------------------------------------
+Upgrade urgency levels:
+LOW:      No need to upgrade unless there are new features you want to use.
+MODERATE: Program an upgrade of the server, but it's not urgent.
+HIGH:     There is a critical bug that may affect a subset of users. Upgrade!
+CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
+SECURITY: There are security fixes in the release.
+--------------------------------------------------------------------------------
 
-There is no release notes for this branch, it gets forked into another branch
-every time there is a partial feature freeze in order to eventually create
-a new stable release.
 
-Usually "unstable" is stable enough for you to use it in development environments
-however you should never use it in production environments. It is possible
-to download the latest stable release here:
+================================================================================
+Valkey 8.1.0 RC1  -  Released Thu 11 Feb 2025
+================================================================================
+Upgrade urgency LOW: This is the first release candidate of Valkey 8.1, with
+performance improvements, extended observability and cluster improvements and different bug fixes. 
+It includes a new implementation of the Valkey dictionary which is more memory and cache efficient, 
+better performance for encryption in transit, reduced replication overhead by offloading work to I/O threads,
+faster failover support in cluster mode, major improvements to the active defrag process to reduce the impact on command processing
+and different API changes for improved usability. 
+Valkey now supports new new check-and-set feature for native STRINGs.
+In this release Valkey also supports CMake build system for enhanced building options and better IDE integration. 
 
-    https://valkey.io/download/
+API and Interface changes
+=========================
+* Introduce bgsave cancel (#757)
+* Add paused_reason to INFO CLIENTS (#1564)
```

Comment:
it's better put it into `Observability and Monitoring changes` with `#1519` together